get_zone_eflh.md

```diff
@@ -0,0 +1,41 @@
+# get_zone_eflh
+
+**Description:** provides the equivalent full load hours of the zone.  Equivalent full load hours are defined as: any hour where the occupancy fraction is greater than 5% AND the HVAC system is in occupied mode.  For this function, we are recognizing the HVAC system as being in occupied mode if ANY of the HVAC systems serving the zone are in occupied mode.
+**Applicability Note:** hvac_system.fan_system.operation_schedule is being used as the HVAC operation schedule.  Therefore, this check will not work for radiant systems or other systems that do not include a fan
+
+**Inputs:** 
+- **RMD**
+- **zone**
+
+**Returns:**  
+- **flh**: a number equal to the total equivalent full load hours for the year
+ 
+**Function Call:**
+- **get_list_hvac_systems_associated_with_zone**
+
+## Logic:
+- create a list to hold information about the occupancy of each space: `people_info_for_spaces = []`
+- create a variable for the total number of zone occupants: `total_zone_occupants = 0`
+- loop through the spaces in the zone collecting information about occupancy: `for space in zone.spaces:`
+  - append the information about the occupancy to the list: `people_info_for_spaces.append([space.number_of_occupants, space.occupant_multiplier_schedule.hourly_values])`
+  - add the occupants of this space to the total number of zone occupants: `total_zone_occupants += space.number_of_occupants`
+- get the list of the hvac systems associated with the zone: `hvac_systems_list = get_list_hvac_systems_associated_with_zone(RMD,zone.id)`
+- create an index to which each full load hour will be added: `flh = 0`
+- now calculate flh for each hour of the year: `for hour in range(8760):`
```

Comment:
This range will need to be updated with the calendar object (`is_leap_year`).

Maybe we just need to get an occupancy schedule and loop over the values instead of using a `range` since RMD only supports hourly schedule (in RCT we decided to not deal with event-based schedule for now).

Reply:
this brings up the question about if the RMD is for a model that was simulated for less than a year?  I've updated to reference the number of values in any one of the occupant schedules.

Reply:
I am not sure such case could happen (theoretically) but almost all standards require for a full year simulation.